heroku_connect/models.py

```diff
@@ -289,3 +301,23 @@ class Meta(TriggerLogAbstract.Meta):
         db_table = '{schema}"."_trigger_log_archive'.format(schema=settings.HEROKU_CONNECT_SCHEMA)
         verbose_name = _('Trigger Log (archived)')
         verbose_name_plural = _('Trigger Logs (archived)')
+
+    @transaction.atomic
+    def redo(self):
+        """
+        Re-apply the change recorded in this trigger log.
+
+        .. seealso:: :meth:`.TriggerLog.redo`
+        """
+        trigger_log = self._copy_to_trigger_log(state=TRIGGER_LOG_STATE['NEW'])
+        trigger_log.save()
+        self.state = TRIGGER_LOG_STATE['REQUEUED']
+        self.save(update_fields=['state'])
```

Comment:
`force_insert=True`

Reply:
This line is (and should be) an update: the `state` of the row in the archive table gets changed after its data was copied to the live table. I'll include the flag in the `trigger_log.save()` above and improve the documentation, to make it clear what's going on here.